package.json

```diff
@@ -4,87 +4,85 @@
-    "@grafana/runtime": "10.2.0",
-    "@grafana/tsconfig": "^1.2.0-rc1",
-    "@grafana/ui": "10.2.0",
-    "@swc/core": "1.3.38",
-    "@swc/helpers": "0.4.14",
-    "@swc/jest": "^0.2.29",
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
-    "@testing-library/user-event": "^14.5.1",
+    "@babel/core": "^7.21.4",
+    "@grafana/aws-sdk": "0.3.4",
+    "@grafana/e2e": "10.4.4",
+    "@grafana/e2e-selectors": "10.4.4",
+    "@grafana/eslint-config": "^7.0.0",
+    "@grafana/tsconfig": "^1.3.0-rc1",
+    "@swc/core": "^1.3.90",
+    "@swc/helpers": "^0.5.11",
+    "@swc/jest": "^0.2.36",
+    "@testing-library/jest-dom": "6.4.6",
+    "@testing-library/react": "16.0.0",
+    "@testing-library/user-event": "^14.5.2",
     "@types/glob": "^8.0.0",
-    "@types/jest": "^27.4.1",
-    "@types/lodash": "latest",
-    "@types/node": "^17.0.19",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
-    "copy-webpack-plugin": "^10.0.0",
-    "cspell": "6.13.3",
-    "css-loader": "6.7.1",
-    "cypress": "7.7.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-jsdoc": "^46.7.0",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.26.1",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-webpack-plugin": "^3.1.1",
-    "fork-ts-checker-webpack-plugin": "^7.2.0",
-    "glob": "^8.0.3",
+    "@types/jest": "^29.5.0",
+    "@types/lodash": "^4.17.5",
+    "@types/node": "^20.8.7",
+    "@types/react-router-dom": "^5.2.0",
+    "copy-webpack-plugin": "^12.0.2",
+    "cspell": "^8.8.4",
+    "css-loader": "^7.1.2",
+    "cypress": "9.5.1",
+    "eslint-plugin-deprecation": "^3.0.0",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-webpack-plugin": "^4.0.1",
+    "fork-ts-checker-webpack-plugin": "^9.0.2",
+    "glob": "^10.2.7",
     "identity-obj-proxy": "3.0.0",
-    "jest": "27.5.0",
-    "prettier": "^2.5.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "prettier": "^3.3.2",
     "react-select-event": "^5.3.0",
     "replace-in-file-webpack-plugin": "^1.0.6",
-    "sass": "1.55.0",
-    "sass-loader": "13.1.0",
-    "style-loader": "3.3.1",
-    "swc-loader": "^0.1.15",
-    "ts-node": "^10.5.0",
-    "tsconfig-paths": "^3.12.0",
-    "tslib": "^2.3.0",
-    "typescript": "^4.4.0",
-    "webpack": "5.76.0",
-    "webpack-cli": "^4.9.2",
+    "sass": "1.77.5",
+    "sass-loader": "14.2.1",
+    "style-loader": "4.0.0",
+    "swc-loader": "^0.2.3",
+    "ts-node": "^10.9.1",
+    "tsconfig-paths": "^4.2.0",
+    "typescript": "5.4.5",
+    "webpack": "^5.86.0",
+    "webpack-cli": "^5.1.4",
     "webpack-livereload-plugin": "^3.0.2"
   },
   "resolutions": {
-    "@grafana/e2e-selectors": "10.0.0",
-    "debug": "2.6.9",
+    "braces": "3.0.3",
```

Comment:
do we use all of these dependencies directly?

Reply:
no, they're deps of other packages.